### ISSUE TITLE
KelbowVisualizer: The letters in the Legend Label are smushed together (missing spacing)

### ISSUE THREAD

**Issue by lwgray** [DistrictDataLabs/yellowbrick#1058]
**Describe the issue**
In the Kelbow visualizer I found that the Spacing between letters in the legend are absent. I also found this to be the case in the example visualization for Kelbow in both the latest and develop version of the docs.  See image below
**To Reproduce**

```python
from sklearn.datasets import make_blobs
from sklearn.preprocessing import StandardScaler
from sklearn.cluster import KMeans

from yellowbrick.cluster import KElbowVisualizer

# Generate sample data
centers = [[1, 1], [-1, -1], [1, -1]]
X, labels_true = make_blobs(n_samples=1500, centers=centers, cluster_std=0.3,
                            random_state=0)

Z = StandardScaler().fit_transform(X)
viz = KElbowVisualizer(KMeans())

viz.fit(Z)
viz.show()
```

**Dataset**
Did you use a specific dataset to produce the bug? Where can we access it?
yes... generated in the above code 

**Expected behavior**
I expect to see "elbow at k = 3, score = 264.727"

**Desktop (please complete the following information):**
 - OS: [e.g. macOS] MacOS
 - Python Version [e.g. 2.7, 3.6, miniconda] 3.7
 - Yellowbrick Version [e.g. 0.7] 1.1

**Additional context**
![image](https://user-images.githubusercontent.com/1000117/79055299-fe423580-7c19-11ea-966b-52db9516eb60.png)

@DistrictDataLabs/team-oz-maintainers
